lib/graphql/query/context.rb

```diff
@@ -309,6 +309,3 @@ def list_of_non_null_items?(type)
     end
   end
 end
-
-
-GraphQL::Schema::Context = GraphQL::Query::Context
```

Comment:
:+1: Looks like orphaned code from my first try at custom context classes, thanks for the cleanup.